lib/@uncharted.software/stories-facets/src/main.js

```diff
@@ -170,6 +172,56 @@ Facets.prototype.highlight = function(simpleGroups, isQuery) {
 };
 
 /**
+ * Creates badges for the provided facets.
+ *
+ * @method createBadges
+ * @param {Array} simpleGroups - An array containing the group keys and facet values to create badges for.
+ * @param {boolean=} isQuery - Optional parameter to define if the subgroup is a query, if not specified the method will try to auto-detect the group's type.
```

Comment:
Just nitpicking: Optional parameter to define if the subgroup is a query; if not specified, the method will try to auto-detect the group's type.